src/stores/mainVehicle.ts

```diff
@@ -153,6 +154,19 @@ export const useMainVehicleStore = defineStore('main-vehicle', () => {
   function disarm(): void {
     mainVehicle.value?.disarm()
   }
+  /**
+   * Takeoff the vehicle
+   */
+  function takeoff(): void {
+    mainVehicle.value?.takeoff()
+  }
+
+  /**
+   * Land the vehicle
+   */
+  function land(): void {
+    mainVehicle.value?.land()
+  }
```

Comment:
I have mixed feelings with such functions on mainVehicle, that's not a blocker, but just something for us to have in mind for future rework.

The idea of cockpit is to be generic, the concept of takeoff and landing applies mostly to aerial vehicles, having such functions here in mainVehicle is a bit complicated.

I would recommend in the future for us to have a generic callback map that the vehicle could specify such "logic". We need to put more thought on it, there is now no way for us to check if the vehicle does support such functions.

Reply:
Yep. Same thoughts.

But as we don't have discussed yet how to go on that, I think we should just add that now and discuss changes after.